src/applications/caregivers/definitions/content.js

```diff
@@ -67,3 +73,22 @@ export const PrivacyPolicy = () => (
     .
   </p>
 );
+
+export const SecondaryCaregiverCopy = ({ label }) => {
```

Comment:
Moved this copy into the content file with the rest of the signature box content